app/models/user.rb

```diff
@@ -1,4 +1,10 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable,
```

Comment:
Hi @EmiLoKiJu

The user is still automatically logged in after signing up. As requested by the previous reviewer, the user must confirm the email first before they are logged in.

- Kindly consider changing the redirect path after signing up and ensure that the app redirects to the login page and the user can only log in after the confirmation link has been clicked.

Reply:
![image](https://github.com/EmiLoKiJu/BlogApp/assets/84760694/36a8629e-0dce-49a9-a8b2-73c7506c4490)

I have checked my app and I'm not logged in after creatin a new account.